pkg/scaling/scaledjob/scale_metrics_test.go

```diff
@@ -0,0 +1,205 @@
+package scaledjob
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/go-playground/assert/v2"
+	"github.com/golang/mock/gomock"
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+	mock_scalers "github.com/kedacore/keda/v2/pkg/mock/mock_scaler"
+	"github.com/kedacore/keda/v2/pkg/scalers"
+	"k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/metrics/pkg/apis/external_metrics"
```

Comment:
```suggestion
	"github.com/go-playground/assert/v2"
	"github.com/golang/mock/gomock"
	"k8s.io/api/autoscaling/v2beta2"
	"k8s.io/apimachinery/pkg/api/resource"
	"k8s.io/client-go/tools/record"
	"k8s.io/metrics/pkg/apis/external_metrics"
	
	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
	mock_scalers "github.com/kedacore/keda/v2/pkg/mock/mock_scaler"
	"github.com/kedacore/keda/v2/pkg/scalers"
```

Minor nit, we prefer to have `github.com/kedacore` related imports in a separate block :)